scripts/contentScript.js

```diff
@@ -666,7 +808,62 @@ const orchestrator = (tabUrl, websiteUrl, userId) => {
 				}
 			}
 		}
-	})
+
+		if (urlObj[2] === 'gitlab.com') {
+
+			if(urlObj.length === 4 && !urlObj[3]){
+				//TODO: need repo_owner from userID
+								
+
```

Comment:
You don't, actually. We have discussed this on Slack in detail, I believe. We just need to handle this in the 3 URLs that you identified:

1. https://gitlab.com (homepage; which is the condition you have already mentioned)
2. https://gitlab.com/dashboard/projects (all the projects are already listed here as `repo_owner/repo_name` )
3. https://gitlab.com/repo_owner (this is similar to the GitHub org pages; can use a similar API call)

Reply:
LOGAF: 2/10 (I don't mind if you fix this in the next PR)

Reply:
I'll do this in the next PR